020-Connect/010-Postgres/001-connect.mdx

```diff
@@ -0,0 +1,194 @@
+
+```
+postgresql://ws1234:xau_apikey123456@us-east-1.sql.xata.sh:5432/Games:main
+```
+
+The connection strings for the HTTP access and for the Postgres wire protocol are shown to you when you create a new database, afterwards you can always navigate to the database settings to see the URL construction.
+![Database settings showing connection strings](images/connection-string.png)
+
+To create a new API key, or manage your existing ones, you need to open your [account settings](https://app.xata.io/settings).
+
+Using psql to connect to your database, based off the previously constructed connection string with the example values:
+
+```bash
+psql "postgresql://ws1234:xau_apikey123456@us-east-1.sql.xata.sh:5432/Games:main"
+
+Games:main=>
+```
+
+## Supported Versions
+
+You can check the Postgres version with the following command:
+
+```bash
+psql "postgresql://[Workspace Id]:[API Key]@[Region].sql.xata.sh:[Port]/[Database]:[Branch]" -c "select version()"
+```
+
+```sql
+                                                   version
+-------------------------------------------------------------------------------------------------------------
+ PostgreSQL 15.4 on aarch64-unknown-linux-gnu, compiled by aarch64-unknown-linux-gnu-gcc (GCC) 9.5.0, 64-bit
+(1 row)
+```
+
+### Shared cluster
+
+Version: `15.4`
+
+In a shared cluster, you operate in a multi tenant environment, sharing the same Postgres version. Minor version upgrades are continuously updated.
+If you want control of the version, you should consider a dedicated cluster.
+
+### Dedicated cluster
+
+You can select which version you want, currently supported versions in all [available regions](/docs/getting-started/available-regions):
+
+- 15.4
+- 15.5
+- 14.10
+- 14.9
+
+You can control the maintenance windows and the version upgrades yourself.
+
+## Extensions
+
+The available, out-of-the-box extensions are:
+
+- list
+
+If you are on a dedicated cluster you can install extensions yourself, please consult the [Postgres documentation](https://www.postgresql.org/docs/current/sql-createextension.html) for a how to.
+
+## Supported Statements
+
+Xata offers different approaches to Postgres, from SQL over HTTP to pure Postgres over the wire protocol to a shared or dedicated cluster.
+Each come with a security scope, to provide isolation in a multi tenant environment.
+
+|                   | FUNCTIONs | DQL statements | (some) DML statements | DDL statements |
+| ----------------- | --------- | -------------- | --------------------- | -------------- |
+| SQL over HTTP     | N         | Y              | Y                     | Y              |
+| Shared cluster    | N         | Y              | Y                     | Y              |
+| Dedicated cluster | Y         | Y              | Y                     | Y              |
+
+The table below, is providing a more granularity picture of the supported statements.
+
+|                         | SQL over HTTP | Shared Cluster | Dedicated Cluster |
```

Comment:
The distinction of SQL over HTTP and Shared/Dedicated cluster isn't very clear, I suppose this means wire protocol? Maybe rename to `Wire on Shared Cluster` and `Wire on Dedicated Cluster`

Reply:
Funnily enough, I was thinking exactly the same as I was curating the table. I tried to address it in

> Xata offers different approaches to Postgres, from SQL over HTTP to pure Postgres over the wire protocol to a shared or dedicated cluster.

But, likely needs more of an explicit call-out. Thanks Kostas

Reply:
addressed